File: bench/fib.py
import sys
sys.setrecursionlimit(100000)


def fib(n):
    if n < 2:
        return n
    return fib(n - 1) + fib(n - 2)


def sum_to(n):
    acc = 0
    for i in range(1, n + 1):
        acc += i * i % 1000003
    return acc


print(fib(27))
print(sum_to(2000000))

File: bench/go/fib.go
// 调用密集：递归 fib（内联与调用约定的账）
package main

func fib(n int) int {
	if n < 2 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() { println(fib(32)) }
